### PR TITLE
Minor updates to docs, version bumps, and some added tips.

### PR DESCRIPTION
**Pull request by GoodEn0ugh**
Nothing huge in here, just some minor grammatical fixes, formatting, a version bump, and a few added tips.

### CODE REVIEW

**Comment by GoodEn0ugh** [comment]
Added in the initial arrowd docs as well, so this PR is growing a bit :)

File: docs/how-to/maintain-quiver/backup-and-restore.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ Close Quiver.
 1. Select **File->Export all private keys**. All your private keys are displayed in a new window.
 1. Store them securely.
 
+!!! Tip "Save keys as a file"
+
+    If you'd like to save the keys as a file, simply select **Save** and place them in a secure location.
```
Comment (geronimoiceberg) [inline docs/how-to/maintain-quiver/backup-and-restore.md]: Let's use this: "If you want to save the keys as a file, click **Save**, and store them in a secure place.

File: docs/how-to/manage-transactions/receive-send-arrow.md [hunk 2]
```diff
@@ -23,4 +23,8 @@ To send Arrow, you must have an address with an Arrow balance.
         Select a recipient's address by using [the address book](../maintain-addresses/create-address-book.md).
 
 1. In the **Amount** field, specify the amount of Arrow to send.
+
+    !!! Tip
+        If you'd like to send the entire balance of an address, check the **max available** box before sending.
```
Comment (geronimoiceberg) [inline docs/how-to/manage-transactions/receive-send-arrow.md]: Let's use this: "If you want to send the entire balance of an address, check the **Max Available** box."  No quotation marks.

File: docs/how-to/maintain-quiver/backup-and-restore.md (hunk 1 above)
Comment (GoodEn0ugh) [reply docs/how-to/maintain-quiver/backup-and-restore.md]: Looks good to me.

File: docs/how-to/manage-transactions/receive-send-arrow.md (hunk 2 above)
Comment (GoodEn0ugh) [reply docs/how-to/manage-transactions/receive-send-arrow.md]: Looks more clean :) I like it.

File: docs/how-to/manage-transactions/receive-send-arrow.md [hunk 3]
```diff
@@ -15,6 +15,10 @@ Before you can receive Arrow, you must have [an Arrow address](../maintain-addre
 
 To send Arrow, you must have an address with an Arrow balance.
 
+!!! Warning "Sending many UTXOs in one batch"
+
+    If you have many distinct UTXOs (unspent transactions) in your wallet and attempt to spend many of them, it may take some time to compute the resulting transaction.
```
Comment (geronimoiceberg) [inline docs/how-to/manage-transactions/receive-send-arrow.md]: Let's use this: "If you have many unspent transactions (UTXOs) in your wallet and you try to spend many of them, processing the transaction might take a long time."

Comment (GoodEn0ugh) [reply docs/how-to/manage-transactions/receive-send-arrow.md]: I'm cool with the change, but "processing" might come across as it being the network that delays it.

Just want to make sure it's clear that the wallet itself has to compute the Tx which can be a lengthy process when combining many outputs.

Comment (geronimoiceberg) [reply docs/how-to/manage-transactions/receive-send-arrow.md]: Ah. I see. Thanks for clarifying. In this case, how about, "computing the transaction might take a long time." What do you think?

Comment (GoodEn0ugh) [reply docs/how-to/manage-transactions/receive-send-arrow.md]: Like that a lot better.

File: docs/how-to/run-arrowd/compile-arrowd.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+description: Compile arrowd
+<!--- END of page meta data -->
+
+# Compile arrowd
+
+## **_Getting Started_**
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/compile-arrowd.md]: Let's remove line 6

File: docs/how-to/run-arrowd/compile-arrowd.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+### Recommended System Specs:
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/compile-arrowd.md]: Let's change this line to ## System requirements

File: docs/how-to/run-arrowd/compile-arrowd.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+2 or more CPU threads  
+64-bit OS  
+4GB+ RAM  
+20GB+ storage  
+
+## **Building for Linux from source**
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/compile-arrowd.md]: Remove the sets of asterisks at the beginning and end of the title

File: docs/how-to/run-arrowd/compile-arrowd.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+./zcutil/fetch-params.sh
+```
+
+### **Run arrowd**
+
+```
+./src/arrowd
+```
+
+## **Building for MacOS from source**
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/compile-arrowd.md]: Remove the sets of **

File: docs/how-to/run-arrowd/compile-arrowd.md (hunk 6 above)
Comment (geronimoiceberg) [reply docs/how-to/run-arrowd/compile-arrowd.md]: Let's say "Build for Linux from source"

File: docs/how-to/run-arrowd/compile-arrowd.md (hunk 7 above)
Comment (geronimoiceberg) [reply docs/how-to/run-arrowd/compile-arrowd.md]: Let's say "Build for MacOS from source"

File: docs/how-to/run-arrowd/compile-arrowd.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+
+### Install dependencies
+
+```
+xcode-select --install
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+brew install cmake autoconf libtool automake coreutils pkgconfig gmp wget
+brew install gcc5 --without-multilib
+```
+### **Build**
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/compile-arrowd.md]: Remove sets of **

File: docs/how-to/run-arrowd/compile-arrowd.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+```
+git clone https://github.com/Arrowchain/arrow.git arrow
+cd arrow
+./zcutil/build.sh -j$(sysctl -n hw.ncpu)
+```
+
+## **Fetch params**
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/compile-arrowd.md]: Remove sets of **

File: docs/how-to/run-arrowd/compile-arrowd.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+
+``` 
+./zcutil/fetch-params.sh
+ ```
+
+### **Create config file**
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/compile-arrowd.md]: Remove sets of **

File: docs/how-to/run-arrowd/compile-arrowd.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+2. echo "rpcuser={username}" >> ~/Library/Application Support/arrow/arrow.conf
+3. echo "rpcpassword=`head -c 32 /dev/urandom | base64`" >> ~/Library/Application Support/arrow/arrow.conf
+4. echo "rpcallowip=127.0.0.1" >> ~/Library/Application Support/arrow/arrow.conf
+5. echo "addnode=52.90.76.26:7654" >> ~/Library/Application Support/arrow/arrow.conf
+6. echo "addnode=45.77.143.128:7654" >> ~/Library/Application Support/arrow/arrow.conf
+7. echo "addnode=85.15.179.171:7654" >> ~/Library/Application Support/arrow/arrow.conf
+8. echo "addnode=65.100.172.203:7654" >> ~/Library/Application Support/arrow/arrow.conf
+9. echo "addnode=34.204.183.163:7654" >> ~/Library/Application Support/arrow/arrow.conf
+```
+### **Run arrowd**
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/compile-arrowd.md]: Remove sets of **

File: docs/how-to/run-arrowd/configuring-arrowd.md [hunk 12]
```diff
@@ -0,0 +1,39 @@
+description: arrowd configuration
+<!--- END of page meta data -->
+
+# arrowd configuration
+
+## Example arrowd config
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/configuring-arrowd.md]: Let's get rid of line 6

File: docs/how-to/run-arrowd/configuring-arrowd.md [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+
+Below is an example arrow.conf, which can be placed in the proper [default data directory](../troubleshoot/find-data-dir.md) to configure how arrowd operates
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/configuring-arrowd.md]: How about "The `arrow.conf` file configures how arrowd operates. You can put the following example file in the proper [default data directory](../troubleshoot/find-data-dir.md). Replace the variables in brackets with the proper data."

And then add the example inside an example tag, like we're doing for tips and warning and things (!!! Example)

File: docs/how-to/run-arrowd/configuring-arrowd.md [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+# Add seed nodes
+addnode=52.90.76.26:7654
+addnode=45.77.143.128:7654
+addnode=85.15.179.171:7654
+addnode=65.100.172.203:7654
+addnode=34.204.183.163:7654
+```
+
+## Allowing inbound connections
+If you would like to help support the network by seeding the blockchain to other users, you must expose the default p2p port `7654/tcp` to the internet.
```
Comment (geronimoiceberg) [inline docs/how-to/run-arrowd/configuring-arrowd.md]: "If you want to help support the network by seeding the blockchain to other users, expose the default p2p port `7654/tcp` to the internet."

File: docs/how-to/maintain-quiver/backup-and-restore.md (hunk 1 above)
Comment (GoodEn0ugh) [reply docs/how-to/maintain-quiver/backup-and-restore.md]: Done.

File: docs/how-to/run-arrowd/configuring-arrowd.md (hunk 13 above)
Comment (GoodEn0ugh) [reply docs/how-to/run-arrowd/configuring-arrowd.md]: Double check the changes I'm pushing, but I think I've implemented as asked here.

Comment (GoodEn0ugh) [reply docs/how-to/run-arrowd/configuring-arrowd.md]:
```
The arrow.conf file configures how arrowd operates. You can put the following example file in the proper [default data directory](../troubleshoot/find-data-dir.md). Replace the variables in brackets with the proper data.

!!! Example

    {username} - Any username you would like
    {password} - A secure password
    {internal_ip} - Your `arrowd` host's IP address
    {local_subnet} - Your local subnet (or use localhost if you would only like the server itself to be allowed to connect)
    {max_conn} - any integer, the higher the number the more inbound and outbound arrowd servers can connect
```

**Pull request merged**